### PR TITLE
[eclipse/xtext-core#1626] regnerate languages

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext-core#1626] regnerate languages

### CODE REVIEW

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java [hunk 1]
```diff
@@ -83,132 +83,114 @@
...
-		//proteasesite | proteinstab | rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter
+		//octagon | oval |
+		//parallelogram | pentagon | plain | plaintext | point | polygon | primersite | promoter | proteasesite | proteinstab |
+		//rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter
 		public Alternatives getAlternatives() { return cAlternatives; }
 		
 		//octagon
 		public EnumLiteralDeclaration getOctagonEnumLiteralDeclaration_0() { return cOctagonEnumLiteralDeclaration_0; }
 		
-		//"octagon"
```
Comment (ArneDeutsch) [inline org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java:97]: We have a small loss of information here. Guess it is not really an issue. @szarnekow !?

Comment (cdietrich) [reply org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java:97]: is here no node or dont we retrieve it correctly?

**Review by ArneDeutsch — APPROVED** [review]

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java (hunk 1 above)
Comment (szarnekow) [reply org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java:97]: There is no node. The string literal is optional and added during the Xtext linking.

File: org.eclipse.xtext.builder.tests/src-gen/org/eclipse/xtext/builder/tests/services/BuilderTestLanguageGrammarAccess.java [hunk 2]
```diff
@@ -55,16 +55,16 @@
...
 		private final Keyword cRightCurlyBracketKeyword_5 = (Keyword)cGroup.eContents().get(5);
 		
-		//Namespace:
-		//	'namespace' name=QualifiedName '{'
-		//	imports+=Import*
-		//	elements+=NamedElement*
-		//	'}';
+		//Namespace :
+		//    'namespace' name=QualifiedName '{'
+		//        imports+=Import*
```
Comment (szarnekow) [inline org.eclipse.xtext.builder.tests/src-gen/org/eclipse/xtext/builder/tests/services/BuilderTestLanguageGrammarAccess.java:60]: ❤️

File: org.eclipse.xtext.common.types.eclipse.tests/src-gen/org/eclipse/xtext/common/types/xtext/ui/services/RefactoringTestLanguage1GrammarAccess.java [hunk 3]
```diff
@@ -80,8 +80,8 @@ public TerminalsGrammarAccess getTerminalsGrammarAccess() {
 	}
 
 	
-	//Entry Model:
-	//	Model;
+	//Entry returns Model:
```
Comment (szarnekow) [inline org.eclipse.xtext.common.types.eclipse.tests/src-gen/org/eclipse/xtext/common/types/xtext/ui/services/RefactoringTestLanguage1GrammarAccess.java:83]: nice nice nice!

**Review by szarnekow — APPROVED** [review]
I think this is a big improvement.

**Pull request merged**